content/collections.md

```diff
@@ -0,0 +1,189 @@
+---
+title: Collections and Models
+---
+
+After reading this guide, you'll know:
```

Comment:
Do you think this is a good format for an introduction? I like it, but I heard not everyone is into it.